test/integration/deprecated_specs.cc

```diff
@@ -38,3 +42,89 @@ TEST(DeprecatedSpecs, Spec1_2)
   sdf::SDFPtr sdf(new sdf::SDF());
   EXPECT_FALSE(sdf::initFile(filename, sdf));
 }
+
+////////////////////////////////////////////////////
+TEST(DeprecatedElements, CanEmitErrors)
+{
+  sdf::SDFPtr sdf(new sdf::SDF());
+  sdf::init(sdf);
+  auto elem = std::make_shared<sdf::Element>();
+  elem->SetRequired("-1");
+  elem->SetName("testElem");
+  // Change the root of sdf so we can test "testElem" in isolation
+  sdf->Root(elem);
+  auto config = sdf::ParserConfig::GlobalConfig();
+  config.SetWarningsPolicy(sdf::EnforcementPolicy::ERR);
+  sdf::Errors errors;
+  EXPECT_TRUE(sdf::readString(
+      "<sdf version='1.8'><testElem/></sdf>", config, sdf, errors));
+  ASSERT_FALSE(errors.empty());
+  EXPECT_EQ(sdf::ErrorCode::ELEMENT_DEPRECATED, errors[0].Code());
+}
+
+////////////////////////////////////////////////////
+TEST(DeprecatedElements, CanEmitWarningWithErrorEnforcmentPolicy)
+{
+  // Capture sdfwarn output, which is treamed to std::cerr
+  std::stringstream buffer;
+  auto old = std::cerr.rdbuf(buffer.rdbuf());
+
+#ifdef _WIN32
+  sdf::Console::Instance()->SetQuiet(false);
```

Comment:
nit Ew... this seems nigh defective in terms of public contract.

I don't see any indicator in `Console.hh` that indicates this would be necessary?

Reply:
I don't know if tests still fail if we don't disable the output on windows, but that's what the todo says: https://github.com/osrf/sdformat/blob/25004f0bad227cf11854cc208e74be08e8623359/src/Console.cc#L35-L41

Reply:
Added issue: https://github.com/osrf/sdformat/issues/554